data/coalition/lunarium intro.txt

```diff
@@ -0,0 +1,2028 @@
+				accept
+			label lunarium
+			`	"Lucky we are that joined us, you have, Captain <last>. Wanted for years to try some tactics in real combat, I had, but of course, do it here without risking our whole operation, we could not. Hope I do that boost the confidence of our lower ranks, the recent experience we gained will." She asks you to be careful in your travels, and says goodbye, following the others into the city.`
+	on fail
+		"assisting lunarium" --
+
+
+
+mission "Lunarium: Questions"
+	name "Lunarium Meeting"
+	description "Now that you've further helped the Lunarium, head to <destination>, where you were invited to meet with their leaders, who have a special task in mind for you."
+	minor
+	landing
+	source
+		government "Coalition"
+		not planet "Remote Blue"
+	destination "Remote Blue"
+	to offer
+		or
+			"assisted lunarium" == 5
+			and
+				"coalition jobs" >= 80
+				"assisted lunarium" == 4
+			and
+				"coalition jobs" >= 110
+				"assisted lunarium" == 3
+		"reputation: Quarg" >= 0
+		not "Heliarch License: done"
+		not "assisting heliarchy"
+		not "assisting lunarium"
+	to fail
+		or
+			has "Heliarch License: done"
+			"reputation: Quarg" < 0
+	to complete
+		has "Lunarium: Quarg Interview: accepted"
+	on offer
+		require "Jump Drive"
+		conversation
+			`As you prepare to land on <origin>, your monitor starts producing a low hum, stops, then starts again. This keeps up for nearly a minute until a message pops up. "Captain <last>, come to trust you our comrades have, thanks to your consistent help to our cause. Interested we are in your continuous support, so that free the Coalition, together we can. To <destination> you must come, meet with you, our leaders will, so that discuss an important task with them, you may. An available bunk in your ship, you should have."`
+			`	The message cuts off shortly after you finish reading it. Your monitor is back to normal, and no humming sound can be heard. You mark <planet> on your map.`
+				accept
+
+mission "Lunarium: Quarg Interview"
+	name "The Interview"
+	description "Travel to <destination> with Oobat, an Arach diplomat from the Lunarium, to try and speak with the Quarg about their thoughts on the current Coalition."
+	passengers 1
+	landing
+	blocked "You need <capacity> in order to take on the next mission. Return here when you have the required space free."
+	source "Remote Blue"
+	destination "Lagrange"
+	to offer
+		has "Lunarium: Questions: active"
+	on offer
+		conversation
+			`Here on <origin>, the Lunarium leaders are waiting for you. Would you like to meet with them and officially join the Lunarium? Given their cause against the Heliarchs, this would no doubt set you on path to conflict with the Heliarchy.`
+			choice
+				`	(Yes.)`
+				`	(Not yet.)`
+					defer
```

Comment:
```suggestion
				`	(Yes.)`
				`	(Not yet.)`
					defer
				`	(No. I don't want to support the Lunarium at all.)`
```
Open to thoughts on this. I just want to add a definitive choice for anyone who accidentally finds themselves here but was planning to support the Heliarchs.